tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -157,7 +157,7 @@ public function testCreateTableWithForeignKeys()
 
         $table = new \Phinx\Db\Table('ntable', array(), $this->adapter);
         $table->addColumn('realname', 'string')
-              ->addColumn('tag_id', 'integer')
+              ->addColumn('tag_id', 'integer', array('signed' => false))
```

Comment:
Right, so this is exactly what worries me about this PR... by changing the default expectation for PK columns, we suddenly have to worry about every single FK reference too. From my standpoint, this PR drives MySQL adapter further from compatibility with other adapters, by making Phinx less consistent.

Thoughts?